bundles/org.openhab.binding.teslapowerwall/README.md

```diff
@@ -0,0 +1,100 @@
+# TeslaPowerwall Binding
+
+This binding enables the capture of key data from a Tesla Powerwall 2 into openHAB.
+
+## Supported Things
+
+Tesla Powerwall 2
+
+## Discovery
+
+The binding does not support auto discovery.
+
+## Thing Configuration
+
+As a minimum, the IP address is needed:
```

Comment:
```suggestion
As a minimum, the hostname is needed:
```
Should be aligned with the parameter name

Reply:
fixed.